### ISSUE TITLE
LensedPS before lensing

### ISSUE THREAD

**Issue by gvernard** [aymgal/COOLEST#53]
It should be very easy to extend the LensedPS class in order to account for a non-lensed PS. This will be useful for codes like MOLET and Herculens. I can add this.

**Comment by aymgal** [comment]
Yes! I agree this is needed to have unlensed point source supported here. I think I am in favor of having a single, unified `PointSource` light profile, with "parameters" fields `lensed_positions` and `observed_positions` (or similar), and as you propose, an additional class attribute stating if it is lensed or not. This way, unlensed point sources will have only the `observed_positions` filled up. And lensed point sources can either have their lensed or unlensed positions filled up, or both which I think could be useful because one can get directly from the temple both sets of positions.
An alternative is to go more minimalist and have only one `positions` parameters, and depending if the lensed flag is true or not, it contains lensed or unlensed position. But this is more prone to mistakes and misinterpretation in my opinion.

Please go ahead with opening a pull request from a new branch, if you want to have a go at it. That would be great!

I propose that we leave the update of the plotting interface to the next step, after converging on the template interface first.

**Comment by gvernard** [comment]
I created a pull request and will be pushing my updates there soon.
